app/controllers/collection_versions_controller.rb

```diff
@@ -37,23 +37,31 @@ def edit
     @form.prepopulate!
   end
 
+  # rubocop:disable Metrics/MethodLength
+  # rubocop:disable Metrics/AbcSize
   def update
     collection_version = CollectionVersion.find(params[:id])
-    clean_params = collection_params
+    orig_collection_version = collection_version
```

Comment:
Why is this assignment necessary? Both variables point to the sma object?

Reply:
`collection_version` eventually gets re-assigned and we need to send the initial version to `context_form` on line 55. Maybe there is a better way to do this?